static/js/portico/signup.js

```diff
@@ -196,4 +196,9 @@ $(function () {
     $("body").on("click", "#choose_email .choose-email-box", function () {
         this.parentNode.submit();
     });
+
+    $('#id_terms').on('change', function () {
```

Comment:
Watching for the change here is not enough. You need to set the correct value when the page is reloaded as well. For example, create an organization with empty subdomain, then try to create a new organization and fill everything except the subdomain. Then click submit. The browser will reload and shows the error that subdomain is not available. Even though now the accept term is clicked the button would still be shown as disabled since you enabling/disabling the button only during on change even.

![Screenshot from 2020-03-02 18-11-54](https://user-images.githubusercontent.com/7190633/75677685-0cfc0b00-5cb2-11ea-82fc-ab729fccc421.png)